configureBuild.sh

```diff
@@ -176,7 +176,7 @@ setRepository() {
   elif [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" == "${JDK8_CORE_VERSION}" ] && [ "${BUILD_CONFIG[OS_ARCHITECTURE]}" == "armv7l" ] && [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_TEMURIN}" ]]; then
     suffix="adoptium/aarch32-jdk8u";
   elif [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" == "${JDK8_CORE_VERSION}" ] && [ "${BUILD_CONFIG[OS_ARCHITECTURE]}" == "armv7l" ] && [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_HOTSPOT}" ]]; then
-    suffix="adoptium/aarch32-port-jdk8u";
+    suffix="openjdk/aarch32-port-jdk8u";
```

Comment:
Why aren't we using a mirror in Adoptium?

Reply:
For `VARIANT=hotspot` we use the upstream `openjdk` repositories' master branches in the scripts to give a fully "Vanilla" openjdk experience, for `VARIANT=temurin` we use our mirrors.